base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -57,6 +56,7 @@ use tari_utilities::{
     ByteArray,
 };
 
+use super::{base_layer_core_chain_storage_lmdb_hasher, LMDB_STORAGE_HASH_LABEL};
```

Comment:
As per simplification:
```suggestion
use crate::chain_storage::lmdb_db::CoreChainStorageHasherBlake256;
```